torch/distributed/optim/post_localSGD_optimizer.py

```diff
@@ -65,10 +66,31 @@ def __repr__(self):
         return self.optim.__repr__()
 
     def state_dict(self):
-        return self.optim.state_dict()
+        r"""
+        This is same as torch.optim.Optimizer state_dict,
+        but adds an extra entry to record model averager's step to the checkpoint
+        to ensure reload does not cause unnecessary warm up again.
+        """
+        optim_state_dict = self.optim.state_dict()
+        optim_state_dict['step'] = self.averager.step
+        return optim_state_dict
 
     def load_state_dict(self, state_dict):
+        r"""
+        This is same as torch.optim.Optimizer load_state_dict,
```

Comment:
Going off of the previous comment on the docstrings, I personally would write:
```suggestion
        This is same as :class:`torch.optim.Optimizer` `:meth:load_state_dict`,
```

Reply:
Thanks for the example!